provisioning/miniprov/hare_mp/main.py

```diff
@@ -282,7 +282,7 @@ def test_IVT(args):
 def reset(args):
     try:
         rc = 0
-
+        init(args)
```

Comment:
1. `init` may exit. Please refactor `init()` function so it will not stop the whole application unexpectedly. Without it it is dangerous to compose functions. Putting `exit()`s within the functions was initially not the best idea.
2. Please add a comment explaining why this init is required.

Reply:
This is regarding the RESET phase of the mini provisioner.
In motr reset, motr clean up the motr and IO tests generated data. But to restart the cluster, hare init needed to be called.
As a part of motr reset, we need to mkfs and start m0d services, motr wants hare to start it through hare init.
So its not actually cluster start but it is services start.

Reply:
Addressed.